pkg/orchestrator/pod_test.go

```diff
@@ -522,6 +546,22 @@ func getScrubCases() map[string]struct {
 				Env: []v1.EnvVar{{Name: "password", Value: "********"}},
 			},
 		},
+		"sensitive arg": {
+			input: v1.Container{
+				Args: []string{"mysql", "--password", "afztyerbzio1234"},
```

Comment:
Why do we have `Args` looking like another full-fledged command in those tests?
What about the case where command is for instance `["mysql"]` and args contains `["--password", "foo"]` ?